openmetadata-service/src/main/java/org/openmetadata/service/util/EntityUtil.java

```diff
@@ -486,4 +487,22 @@ public static void sortByTagHierarchy(List<Tag> tags) {
     // Sort tags by tag hierarchy. Tags with parents null come first, followed by tags with
     tags.sort(Comparator.comparing(Tag::getFullyQualifiedName));
   }
+
+  /**
+   * This method is used to populate the entity with all details of EntityReference Users/Tools can send minimum details
+   * required to set relationship as id, type are the only required fields in entity reference, whereas we need to send
+   * fully populated object such that ElasticSearch index has all the details.
+   */
+  public static List<EntityReference> getEntityReferences(List<EntityReference> entities, Include include)
+      throws IOException {
+    if (nullOrEmpty(entities)) {
+      return Collections.emptyList();
+    }
+    List<EntityReference> refs = new ArrayList<>();
+    for (EntityReference entityReference : entities) {
+      EntityReference chartRef = Entity.getEntityReference(entityReference, include);
+      refs.add(chartRef);
+    }
+    return refs.isEmpty() ? null : refs;
```

Comment:
refs will always have members because the check on line 498 already checked the list is not empty.